### ISSUE TITLE
silently hangs if recaptcha key has not been set

### ISSUE THREAD

**Issue by techieshark** [t49tran/react-google-recaptcha-v3#34]
Hi, thanks for putting this library together.

I ran into an issue I'll share here for others' benefit. 

The problem was when executing the recaptcha fetch, it just hung. I was expecting either an immediate response or maybe throwing some error.

in App.jsx:

```jsx
import { GoogleReCaptchaProvider } from 'react-google-recaptcha-v3';
<GoogleReCaptchaProvider
  reCaptchaKey={process.env.RECAPTCHA_KEY}
> 
  <Form/>
</GoogleReCaptchaProvider>
```

then in Form component:

```jsx
 refetchCaptchaToken = async () => {
    const { googleReCaptchaProps } = this.props;
    return googleReCaptchaProps.executeRecaptcha('submission');
}
```

but upon calling `refetchCaptchaToken` in my form submission handler, the `await` doesn't return, it just hangs.

---

## Solution

It turns out I was passing undefined as the `reCaptchaKey` to `GoogleReCaptchaProvider`. 
Instead of  `reCaptchaKey={process.env.RECAPTCHA_KEY}` I needed to use `reCaptchaKey={process.env.REACT_APP_RECAPTCHA_KEY}`. 

Still it took some time to debug. Would be great if there was a faster way to notice that error - perhaps throwing an error or console.error logging something?

**Comment by t49tran** [comment]
Hi @techieshark , I will have a look, I think it's a good idea to throw an error when there is no `recaptchaKey` provided.

**Comment by t49tran** [comment]
I added a warning if there is no `recaptchaKey` provided here: https://github.com/t49tran/react-google-recaptcha-v3/commit/43f3cad6b7af557bcc79292dc3d7a38997e1cc3e

This one will be updated with the next release.

**Comment by t49tran** [comment]
https://github.com/t49tran/react-google-recaptcha-v3/tree/v1.5.5 addressed on v1.5.5.
